amr-wind/wind_energy/actuator/disk/Joukowsky.H

```diff
@@ -22,6 +22,10 @@ struct JoukowskyData : public DiskBaseData
     // --- Sorenson 2020 equation 10 constants ----
     amrex::Real root_correction_coefficient{2.335};
     amrex::Real root_correction_exponent{4.0};
+    // --- Sorenson 2022 equation 26 constants ----
```

Comment:
@lawrenceccheung let's add a comment here about the change in sign, and using 0.2 vs 0.8 for the C_t < C_t,r case.

Reply:
Thanks @psakievich, I added some comments explaining those values in the latest commit.